### ISSUE TITLE
Version 1 code and version 2 pretrain model?

### ISSUE THREAD

**Issue by tommy19970714** [makeuptransfer/SCGAN#4]
Awesome paper! It is an excellent result.

I was using this repository as soon as it was released.
It seems that version 2 was released 3 days ago, but the code of version 1 has disappeared from github.
Could you please publish the code of version1 in a branch called version1?

Also, I tried to use version2, but the pretrain model in the readme was the version1 vgg model.
Can you please release the pretrain model of version2?

**Comment by PennyPeng369** [comment]
I tried to run test using version 2 code with pretrained models (G.pth & vgg.pth) downloaded from google drive. The results looks incorrect totally. The pixel value range of transferred image is [-0.038, 0.038].

**Comment by tommy19970714** [comment]
@PennyPeng369 Did you load vgg.pth?
The structure in the VGG model is different between version1 and vesion2, so I couldn't load it.

version2: https://github.com/makeuptransfer/SCGAN/blob/master/models/vgg.py
version1: https://github.com/tommy19970714/SCGAN/blob/master/models/vgg.py

**Comment by PennyPeng369** [comment]
> @PennyPeng369 Were you able to load vgg.pth?
> The structure in the VGG model is different between version1 and vesion2, so I couldn't load it.
> 
> version2: https://github.com/makeuptransfer/SCGAN/blob/master/models/vgg.py
> version1: https://github.com/tommy19970714/SCGAN/blob/master/models/vgg.py
I can load it to run phase test, but cannot run phase train. From my understanding, there are two parts using vgg19. One is in PartStyleEncoder(SCGen.py: line 153 to line 154) to extract features, which is a normal vgg19 provided by torchvision. Another one is to compute vgg loss(SCGAN.py: line 69 to line 71) only used in training phase, which is an author-defined vgg19 with an extra input params `out_keys` and different layer keys.

**Comment by makeuptransfer** [comment]
sorry for that, i will update the another vgg weights of the model

**Comment by makeuptransfer** [comment]
Link updated
